src/coord/src/catalog.rs

```diff
@@ -1888,7 +1889,7 @@ impl<S: Append> Catalog<S> {
                 Err(e) => {
                     return Err(Error::new(ErrorKind::Corruption {
                         detail: format!("failed to deserialize item {} ({}): {}", id, name, e),
-                    }))
+                    }));
```

Comment:
I meant to include this comment in my last review but looks like I forgot to press send, sorry about that. Do you need these formatting changes to pass the formatting checks in CI? If not, would you mind reverting them all. As long as the code passes CI then the formatting is OK, and changes like these can end up obfuscating the git commit history.

Reply:
Yes,i think you are right. We should not to obfuscat git commit hitstory. I find the reason of it is my `IDE`.Its format command is used `built-in format` rather than `rustfmt`. I unified him into `rustfmt`